docs/concepts/policy/pod-security-policy.md

```diff
@@ -554,3 +555,8 @@ specifies which values are allowed for the pod seccomp annotations. Specified as
 a comma-delimited list of allowed values. Possible values are those listed
 above, plus `*` to allow all profiles. Absence of this annotation means that the
 default cannot be changed.
+
+### Sysctl
+
+Controlled via annotations on the PodSecurityPolicy. Refer to the [Sysctl documentation](
```

Comment:
This is great, but would be better as a direct reference. For example, change:
> `https://kubernetes.io/docs/concepts/cluster-administration/sysctl-cluster/#podsecuritypolicy-annotations`

to:
> `/docs/concepts/cluster-administration/sysctl-cluster/#podsecuritypolicy-annotations`